Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -495,6 +495,30 @@ mode would look as follows:
 
 .. _XDP acceleration:
 
+Socket LoadBalancer Bypass in Pod Namespace
```

Comment:
nit: this messes up `_XDP acceleration` tag above. Also, I would drop `Socket` from the title.